### PR TITLE
New formatting and content for Helidon GraphQL SE Server document

### PR DESCRIPTION
**Pull request by spericas**
New formatting and content for Helidon GraphQL SE Server document.

Signed-off-by: Santiago Pericasgeertsen <santiago.pericasgeertsen@oracle.com>

### CODE REVIEW

File: docs/se/graphql.adoc [hunk 1]
```diff
@@ -41,63 +51,70 @@ include::{rootdir}/includes/dependencies.adoc[]
...
+
+<1> Define the GraphQL schema.
+<2> Create a `DataFetcher` to return a list of hellos in different languages.
+<3> Wire up the `DataFetcher` s.
+<4> Generate the GraphQL schema.
+
+== Configuration
 
-The example above deploys a very simple service exposing the `/graphql` endpoint.
+include::{rootdir}/includes/graphql.adoc[]
```
Comment (ljamen) [inline docs/se/graphql.adoc:111]: Is this link complete?

**Review by ljamen — COMMENTED** [review]
LGTM - one question re: link structure.

File: docs/se/graphql.adoc (hunk 1 above)
Comment (spericas) [reply docs/se/graphql.adoc:111]: Yes, it's tested

**Review by tjquinno — APPROVED** [review]
LGTM

**Pull request merged**